src/components/editor/Store.tsx

```diff
@@ -24,6 +24,9 @@ export interface EditorStoreState<T> {
     id: string | null;
     setId: (newVal: EditorStoreState<T>['id']) => void;
 
+    editDraftId: string | null;
```

Comment:
Edit requires a constant reference to the current draft ID. The `id` property of this store is periodically `null` to control the rendering of certain aspects of the _Create_ pages in ways that are not always suitable for _Edit_ pages. Therefore, `editDraftId` is a **temporary** means of providing a constant draft ID reference.